app/screens/settings-screen/security-screen.tsx

```diff
@@ -0,0 +1,196 @@
+          setIsBiometricsEnabled(false)
+        }, (err) => {
+          // unable to remove isBiometricsEnabled
+        })
+    }
+  }
+
+  const handleBiometryAvailabilitySuccess = (isBiometryAvailable: Boolean) => {
+    if (isBiometryAvailable) {
+      authenticate(translate("AuthenticationScreen.setUpAuthenticationDescription"), handleAuthenticationSuccess, handleAuthenticationFailure)      
+    } else {
+      toastShow("Biometric sensor is not available.")
+    }
+  }
+
+  const handleBiometryAvailabilityFailure = () => {
+    toastShow("Please register at least one biometric sensor in order to use biometric based authentication.")
+  }
+
+  const handleAuthenticationSuccess = () => {
+    RNSecureKeyStore.set("isBiometricsEnabled", "1", {accessible: ACCESSIBLE.ALWAYS_THIS_DEVICE_ONLY})
+        .then((res) => {
+          setIsBiometricsEnabled(true)
+        }, (err) => {
+          // unable to store isBiometricsEnabled
+        })
+  }
+
+  const handleAuthenticationFailure = () => {}
+
+  const onPinValueChanged = (value) => {
+    if (value) {
+      navigation.navigate("pin", { screenPurpose: "setPIN" })
+    } else {
+      RNSecureKeyStore.remove("PIN")
+        .then((res) => {
+          RNSecureKeyStore.remove("pinAttempts")
+          setIsPinEnabled(false)
+        }, (err) => {
+          // unable to remove PIN
+        })
+    }
+  }
+
+  return (
+    <Screen style={styles.container} preset="scroll">
+      <View style={styles.settingContainer}>
+        <View style={styles.textContainer}>
+          <Text style={styles.title}>{translate("SecurityScreen.biometricTitle")}</Text>
+          <Text style={styles.subtitle}>{translate("SecurityScreen.biometricSubtitle")}</Text>
+          <Text style={styles.description}>{translate("SecurityScreen.biometricDescription")}</Text>
+        </View>
+        <Switch 
+          style={styles.switch} 
+          value={isBiometricsEnabled} 
+          color={palette.lightBlue}
+          onValueChange={(value => onBiometricsValueChanged(value))} />
+      </View>
+      <View style={styles.settingContainer}>
+        <View style={styles.textContainer}>
+          <Text style={styles.title}>{translate("SecurityScreen.pinTitle")}</Text>
+          <Text style={styles.subtitle}>{translate("SecurityScreen.pinSubtitle")}</Text>
+          <Text style={styles.description}>{translate("SecurityScreen.pinDescription")}</Text>
+        </View>
+        <Switch 
+          style={styles.switch} 
+          value={isPinEnabled} 
+          color={palette.lightBlue}
+          onValueChange={(value => onPinValueChanged(value))} />
+      </View>
+      <View style={styles.settingContainer}>
+        <Button
+          buttonStyle={styles.button}
+          titleStyle={styles.buttonTitle}
+          title={translate("SecurityScreen.setPin")}
+          onPress={() => navigation.navigate("pin", { screenPurpose: "setPIN" })}
+        />
+      </View>
+    </Screen>
+  )
+}
```

Comment:
Line break warning. Hopefully linting will find this when we've integrated `eslint`